examples/xsm.m

```diff
@@ -153,46 +153,35 @@
 
 
 (*left-handed top-quark*)
-ReptL=CreateParticle[{{1,1}},"F"];
+ReptL=CreateParticle[{{1,1}},"F",mq2,"TopL"];
 
 (*right-handed top-quark*)
-ReptR=CreateParticle[{2},"F"];
+ReptR=CreateParticle[{2},"F",mq2,"TopR"];
 
 (*left-handed bottom-quark*)
-RepbL=CreateParticle[{{1,2}},"F"];
+RepbL=CreateParticle[{{1,2}},"F",mq2,"BotL"];
 
 (*light quarks*)
-RepLight=CreateParticle[Range[3,2*4+3],"F"];
+RepLight=CreateParticle[Range[3,2*4+3],"F",mq2,"Light"];
 
 (*Vector bosons*)
-RepGluon=CreateParticle[{1},"V"];
-RepW=CreateParticle[{{2,1}},"V"];
-RepB=CreateParticle[{{3,1}},"V"];
+RepGluon=CreateParticle[{1},"V",mg2,"Gluon"];
+RepW=CreateParticle[{{2,1}},"V",mW2,"W"];
+RepB=CreateParticle[{{3,1}},"V",mB2,"B"];
 
 (*Higgs*)
-RepH = CreateParticle[{1},"S"];
-RepS = CreateParticle[{2},"S"];
+RepH = CreateParticle[{1},"S",ms2,"H"];
+RepS = CreateParticle[{2},"S",ms2,"S"];
 
-
-VectorMass=Join[
-	Table[mg2,{i,1,RepGluon[[1]]//Length}],
-	Table[mW2,{i,1,RepW[[1]]//Length}],
-	Table[mB2,{i,1,RepB[[1]]//Length}]];
-FermionMass=Table[mq2,{i,1,Length[gvff[[1]]]}];
-ScalarMass=Table[ms2,{i,1,Length[gvss[[1]]]}];
-ParticleMasses={VectorMass,FermionMass,ScalarMass};
-(*
-up to the user to make sure that the same order is given in the python code
-*)
-UserMasses={mq2,mg2,mW2,mB2,ms2}; 
-UserCouplings=Variables@Normal@{Ysff,gvss,gvff,gvvv,\[Lambda]4,\[Lambda]3}//DeleteDuplicates;
+(*Light fermions*)
+LightFermions={{2,4,6,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39},"F",mq2,"LightFermions"};
```

Comment:
Maybe you can show the command that tells you which indices should be listed here in a comment?

Reply:
Added in the latest update of the xsm.m example.

Reply:
Okay, nice! Let's also add that to the draft